packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -84,12 +84,8 @@ const updateLocalBootData = (
   return result;
 };
 
-const getCachedOrNull = () => {
-  try {
-    return JSON.parse(storage.getItem(BOOT_LOCAL_KEY));
-  } catch (err) {
-    return null;
-  }
+const getCachedBootOrNull = () => {
```

Comment:
Why did you change this implementation? I think we have the try/catch as in some cases `storage.getItem` may fail

Reply:
This is not mandatory, so I can revert!
I've changed this because maybe we don't need it. According to documentation getItem can return `null` if the key didn't exist. This is making JSON.stringify to throw errors because accepts only `string` values.

Reply:
I think in incognitio it will fail, you can check. if it works no issues

Reply:
It works! 👍

Reply:
I think its better that it throws and we catch whatever, localStorage can sometimes even not be available, let's minimize changes to actual fix of the issue, its easier to review.

Reply:
Yeah, during SSR I think there can be an issue.